src/System.Text.Json/src/System/Text/Json/Serialization/ReadStackFrame.cs

```diff
@@ -234,7 +294,8 @@ public void SetReturnValue(object value)
         }
 
         public bool SkipProperty => Drain ||
-            ReferenceEquals(JsonPropertyInfo, JsonPropertyInfo.s_missingProperty) ||
```

Comment:
So, we changed this condition. Can you explain why?

Reply:
`JsonPropertyInfo.s_missingProperty` now has its `ShouldDeserialize` property set to `false`